src/common/etc/rsyslog.d/xenserver.conf

```diff
@@ -18,6 +18,24 @@ $ActionFileEnableSync off
 # Corosync logs useful things at warning
 if $programname == 'corosync' and $syslogseverity <= 4 then /dev/console
 
+# HTTP disk server backend used by XHA (LINSTOR SR).
+# By default logged in daemon.log (3).
```

Comment:
This may lead to believe that what follows is meant to log it to daemon.log.

```suggestion
# Redirected to a specific log file instead of daemon.log (facility 3).
```

(same principle for lines 28 and 34)